src/test/java/com/oneandone/snmpman/configuration/modifier/CommunityIndexCounter32ModifierTest.java

```diff
@@ -0,0 +1,117 @@
+package com.oneandone.snmpman.configuration.modifier;
+
+import com.oneandone.snmpman.configuration.type.ModifierProperties;
+import org.snmp4j.smi.Counter32;
+import org.snmp4j.smi.OID;
+import org.snmp4j.smi.OctetString;
+import org.snmp4j.smi.Variable;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.testng.Assert.*;
+
+public class CommunityIndexCounter32ModifierTest {
+    private ModifierProperties modifierProperties;
+    private long context1;
+    private long contextValue1;
+    private long context2;
+    private long contextValue2;
+    private final String oid = "1.3.6.1.2.1.17.2.4.0";
+
+    @BeforeTest
+    private void setup() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You may have misspelled a JUnit framework method (setUp or tearDown)](https://www.codacy.com/app/1and1_NDev/snmpman/file/2405103259/issues/source?bid=3233453&fileBranchId=3233458#l25)